lib/hooks/post/rundeck.js

```diff
@@ -0,0 +1,106 @@
+const https = require('https')
+
+const RUNDECK_TOKEN = process.env.RUNDECK_TOKEN
+
+// Registry channels
+const DEV = 'dev'
+const BETA = 'beta'
+const STABLE = 'stable'
+
+const getChannel = version => {
+  if (version.includes('-dev.')) return DEV
+  if (version.includes('-beta.')) return BETA
+  if (version.match(/\d+\.\d+\.\d+/)) return STABLE
+  throw new Error('Unrecognized version channel')
+}
+
+const getInstanceTLD = instance => {
```

Comment:
TLD is confusing as in "toto.cozy.works", it means "works". Here I would use `domain`.